### PR TITLE
[OSJC-281] - Provide support for CPU and memory settings

### PR DESCRIPTION
**Pull request by jeffmaury**
Signed-off-by: Jeff MAURY <jmaury@redhat.com>

### CODE REVIEW

**Comment by jeffmaury** [comment]
@jcantrill @adietish please review

File: src/main/java/com/openshift/internal/restclient/model/ResourceRequirements.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import org.jboss.dmr.ModelNode;
+
+import com.openshift.restclient.model.IResourceRequirements;
+import com.openshift.restclient.model.IResourceSpec;
+
+/**
+ * @author Jeff Maury
+ *
+ */
+public class ResourceRequirements implements IResourceRequirements {
```
Comment (jcantrill) [inline src/main/java/com/openshift/internal/restclient/model/ResourceRequirements.java]: @jeffmaury sorry for the delay... apparently my review comments were not submitted

**Comment by adietish** [comment]
@jeffmaury @jcantrill please consider merging this *AFTER* we bumped to 5.9.0 in https://github.com/openshift/openshift-restclient-java/pull/283 since we dont want to introduce new features in the final sprint towards JBoss Tools 4.5.0.Final.

**Comment by adietish** [comment]
@jeffmaury @jcantrill we're past 5.9.0.Final so there's nothing preventing us from merging this.
@jeffmaury dare to react/act upon @jcantrill's complaint above (so that we can move forward)?

**Comment by jeffmaury** [comment]
I'm still working on this. Please don't merge it yet. I will update today

**Comment by jeffmaury** [comment]
@adietish @jcantrill Ready to be reviewed and merged now

**Comment by adietish** [comment]
[merge]

**Pull request merged**